packages/odyssey-stylelint/src/index.js

```diff
@@ -43,6 +43,6 @@ module.exports = {
     'scss/at-if-closing-brace-newline-after': 'always-last-in-chain',
     'scss/at-if-closing-brace-space-after': 'always-intermediate',
     'selector-type-no-unknown': [ true, { ignore: [ 'custom-elements' ], } ],
-    'liberty/use-logical-spec': true
+    'liberty/use-logical-spec': [ true, { except: ['height', 'min-height', 'max-height', 'width', 'min-width', 'max-width'] } ]
```

Comment:
where are these properties covered?

Reply:
Discussed earlier, but note for posterity: these properties are only required for vertical reading/writing support. Since we currently have no plans to support that mode, allowing the traditional properties will make authoring easier.